packages/config/config/devices/0x0159/smart_plug_16a.json

```diff
@@ -90,25 +91,198 @@
 		},
 		{
 			"#": "30",
-			"$import": "~/templates/master_template.json#state_after_power_failure_prev_off"
+			"label": "Restore on/off status",
+			"description": "After power failure",
+			"valueSize": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Restores last on/off status",
+					"value": 0
+				},
+				{
+					"label": "Does not restore last on/off status",
+					"value": 1
+				}
+			]
 		},
 		{
 			"#": "40",
-			"label": "Power Consumption (Watt) Reporting Threshold",
+			"label": "Power Consumption Reporting Threshold",
+			"description": "Percentage of increase or decrease to be reported.",
 			"unit": "%",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 100,
-			"defaultValue": 20
+			"defaultValue": 20,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				}
+			]
 		},
 		{
 			"#": "42",
-			"label": "Power Consumption (Watt) Reporting Interval",
+			"label": "Power Consumption Reporting Interval",
+			"description": "Time interval for power consumption reports (30 – 32535 seconds).",
 			"unit": "seconds",
 			"valueSize": 2,
 			"minValue": 0,
 			"maxValue": 32535,
-			"defaultValue": 0
+			"defaultValue": 0,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				}
+			]
+		},
+		{
+			"#": "50",
+			"label": "Lower power threshold",
+			"description": "Used in parameter no. 52.",
+			"unit": "W",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 4000,
+			"defaultValue": 30
+		},
+		{
+			"#": "51",
+			"label": "Upper power threshold",
+			"description": "Used in parameter no. 52.",
+			"unit": "W",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 4000,
+			"defaultValue": 50
+		},
+		{
+			"#": "52",
+			"label": "Associated Devices Action",
+			"description": "In case of exceeding defined power values",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 6,
+			"defaultValue": 6,
+			"options": [
+				{
+					"label": "function inactive",
+					"value": 0
+				},
+				{
+					"label": "turn on, once the power drops below lower power threshold",
```

Comment:
Make this label "Turn on once power drops below threshold set in parameter 50. Do the same for the others. And apply the style guide. These should be Sentence case.